optaweb-employee-rostering-backend/src/main/java/org/optaweb/employeerostering/domain/shift/Shift.java

```diff
@@ -46,6 +51,8 @@
     private OffsetDateTime startDateTime;
     @NotNull
     private OffsetDateTime endDateTime;
+    @Transient
+    private Long lengthInHours = null;
```

Comment:
I am guessing you are caching `startDateTime.until(endDateTime, ChronoUnit.HOURS);`? Is that operation expensive enough to justify this?

Reply:
I'll remove it. I haven't done enough benchmarking to show that this is a problem.